backends/tfhe-cuda-backend/cuda/include/integer/integer.h

```diff
@@ -665,5 +665,36 @@ void cuda_integer_signed_scalar_div_radix_kb_64(
 void cleanup_cuda_integer_signed_scalar_div_radix_kb_64(
     void *const *streams, uint32_t const *gpu_indexes, uint32_t gpu_count,
     int8_t **mem_ptr_void);
+
+uint64_t scratch_integer_unsigned_scalar_div_rem_radix_kb_64(
+    void *const *streams, uint32_t const *gpu_indexes, uint32_t gpu_count,
+    int8_t **mem_ptr, uint32_t glwe_dimension, uint32_t polynomial_size,
+    uint32_t lwe_dimension, uint32_t ks_level, uint32_t ks_base_log,
+    uint32_t pbs_level, uint32_t pbs_base_log, uint32_t grouping_factor,
+    uint32_t num_blocks, uint32_t message_modulus, uint32_t carry_modulus,
+    PBS_TYPE pbs_type, bool allocate_gpu_memory, bool is_divisor_power_of_two,
+    bool log2_divisor_exceeds_threshold, bool multiplier_exceeds_threshold,
+    uint32_t num_scalar_bits_for_div, uint32_t num_scalar_bits_for_mul,
+    uint32_t ilog2_divisor, uint64_t divisor, bool allocate_ms_array);
+
+void cuda_integer_unsigned_scalar_div_rem_radix_kb_64(
+    void *const *streams, uint32_t const *gpu_indexes, uint32_t gpu_count,
+    CudaRadixCiphertextFFI *quotient_ct, CudaRadixCiphertextFFI *remainder_ct,
+    int8_t *mem_ptr, void *const *ksks,
+    uint64_t const *decomposed_scalar_for_div,
+    uint64_t const *decomposed_scalar_for_mul,
+    uint64_t const *has_at_least_one_set_for_div,
+    uint64_t const *has_at_least_one_set_for_mul,
+    const CudaModulusSwitchNoiseReductionKeyFFI *ms_noise_reduction_key,
+    void *const *bsks, uint32_t num_scalars_for_div,
+    uint32_t num_scalars_for_mul, bool multiplier_exceeds_threshold,
+    bool is_divisor_power_of_two, bool log2_divisor_exceeds_threshold,
```

Comment:
These arguments need some work. Do we really need all of them? I don't see a reason to add `is_divisor_power_of_two`, `ilog2_divisor`. We can compute those inside the backend.